### ISSUE TITLE
Staging Review finding: Focus management needed for page transitions under Power of Attorney Requests

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104052]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

- When following the navigation link to [/representative/poa-requests](https://staging.va.gov/representative/poa-requests?status=pending&sort=created_at_asc), keyboard focus stays on the link.
- When moving between the Pending and Processed tabs, keyboard focus is lost.
- When following a link to view a request, keyboard focus is lost.
- When submitting a decision on a POA request, keyboard focus is lost.

This has two major consequences for accessibility:
- For sighted keyboard-only users, the viewport updates to show that a change has occurred, but pressing [tab] may take you to an unexpected location on the page. This is disorienting, and it's a huge pain to shift-tab your way back up the page to where you wanted to be in the first place.
- For screen reader users who don't get the visual cue of the page change, nothing is announced by the screen reader with the page transition. Silence can be disorienting and confusing, and users may incorrectly guess that their submission didn't go through.

## Recommended action

This is one of those things that you get out of the box from traditional web pages, which resets focus every time the browser reloads. For an SPA, you have to be a little more proactive in setting focus programmatically on page transitions or else it can get lost.

On these page transitions, keyboard focus should go to the next logical step in the flow (eg. the next [tab] stop makes sense for what a user will want to do), and/or whose text provides clear information on what just happened or where you are. Often these are H1s, H2s, or alerts that describe the outcome of an action.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jquispe-oddball** [comment]
@nihil2501 we spoke about fixing the current loader on the site. I know it was v1 ticket, but prioritizing it for v0 so we can get unblocked for this accessibility ticket. Instead of refreshing the whole page under the nav for /poa-requests, I want to loader to show up under the tabs (as discussed in eng chat a few weeks ago). 

@briandeconinck @shiragoodman on the details page, after the user submits a form, where is the focus expected to land on? the next link on the footer? thanks!

**Comment by jquispe-oddball** [comment]
On load, we no longer lose focus when switching tabs. On card select, the focus goes to the header. If you tab out of the header into the next focusable item, you will not be able to focus on the header. See screenshots. @candilemoine ticket can be closed

<img width="708" alt="Image" src="https://github.com/user-attachments/assets/03fdfc53-d644-4f16-b689-d5246e97b3b7" />
<img width="699" alt="Image" src="https://github.com/user-attachments/assets/6a07e679-9819-4827-9b5e-e223fb2f0c5e" />
